packages/hash/shared/src/createProseMirrorState.ts

```diff
@@ -42,6 +42,11 @@ export const createProseMirrorState = ({
       );
       return true;
     },
+    // execCommand is flagged as depecrated but it seems that there isn't a viable alternative
+    // to call the undo and redo default browser actions (https://stackoverflow.com/a/70831583)
+    // After the collab rework this should be replaced with a proper implementation
+    "Mod-z": () => document.execCommand("undo"),
+    "Mod-y": () => document.execCommand("redo"),
```

Comment:
How about adding ctrl-z / ctrt-shift-z / cmd-shift-z too?

Reply:
I think `ctrl-z` would be covered with `mod` - but would be good to get `mod+shift+z` on here too in order to be a bit more intuitive.

Reply:
Mod-shift-z would is the MacOS shortcut for redo right?

Reply:
Yeah generally I think, though chrome uses cmd+y I think.

Reply:
Can any of you test if this works of mac please?